.editorconfig

```diff
@@ -1,14 +1,20 @@
+# editorconfig.org
+
 root = true
 
-[*.py]
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
 indent_style = space
+insert_final_newline = true
 trim_trailing_whitespace = true
-indent_size = 4
-end_of_line = LF
```

Comment:
@XhmikosR as far as you know, will this change on L7 affect Windows users?

Reply:
@XhmikosR Everything else LGTM.

Reply:
It will still be LF inherited from the first block like it was before

Reply:
@XhmikosR Duh! Of course. Thanks.